internal/target/incus.go

```diff
@@ -250,6 +250,38 @@ func (t *InternalIncusTarget) SetPostMigrationVMConfig(i migration.Instance, all
 	apiDef.Config["volatile.uuid"] = i.UUID.String()
 	apiDef.Config["volatile.uuid.generation"] = i.UUID.String()
 
+	// Apply CPU and memory limits.
+	apiDef.Config["limits.cpu"] = fmt.Sprintf("%d", i.CPU.NumberCPUs)
+	if i.Overrides != nil && i.Overrides.NumberCPUs != 0 {
+		apiDef.Config["limits.cpu"] = fmt.Sprintf("%d", i.Overrides.NumberCPUs)
+	}
+
+	apiDef.Config["limits.memory"] = fmt.Sprintf("%dB", i.Memory.MemoryInBytes)
+	if i.Overrides != nil && i.Overrides.MemoryInBytes != 0 {
+		apiDef.Config["limits.memory"] = fmt.Sprintf("%dB", i.Overrides.MemoryInBytes)
+	}
+
+	// Set UEFI and secure boot configs
+	if i.UseLegacyBios {
+		apiDef.Config["security.csm"] = "true"
+	} else {
+		apiDef.Config["security.csm"] = "false"
+	}
+
+	if i.SecureBootEnabled {
```

Comment:
Secureboot isn't possible if under legacy bios. May be best to put secureboot handling under the else branch above.

Reply:
And ensure that secureboot=false when setting csm=true

Reply:
Done.